block_iagora.php

```diff
@@ -55,9 +55,49 @@ public function get_content() {
         } else {
             $this->content->text = $this->generate_chat_content($copilotendpointurl);
         }
+
         return $this->content;
 
     }
+    /**
```

Comment:
```suggestion
    }

    /**
```
Could we add a newline here to visually separate these two methods?